backend/controllers/adminControllers.js

```
@@ -0,0 +1,192 @@
+    session.startTransaction();
+
+    // 🔍 Find vendor request
+    const request = await VendorRequest.findById(requestId).session(session).exec();
+    if (!request) {
+      await session.abortTransaction();
+      session.endSession();
+      return res.status(404).json({ message: "Request not found" });
+    }
+
+    const userId = request.requesterID;
+    let updateFields = { pendingStatus: "not_pending" };
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'updateFields' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)